serverless.yml

```diff
@@ -10,6 +10,7 @@ provider:
   stage: dev
   region: us-east-1
   environment:
+    ENV: ${self:provider.stage}
```

Comment:
@SavidyaSathyani  @singhabahu  Any reasons why we are using the variable name as "ENV" ?. It might be confusing when we use "process.env.ENV". Better to use same as the provider name "STAGE".

Reply:
Yeah I have to agree, please remove this implementation and update the commit with the logger

Reply:
I didn’t see any variable specific to environment in the process.env. Thats why I added the stage as ENV so it get include in the process.env. And if we are to remove the ENV then we won’t be able to manage environments of the application. If the application doesn’t have any environment specific variables we can completely remove the config file implementation. so do you want me to completely remove this config file or just this ENV variable?